sljit_src/sljitLir.c

```diff
@@ -2228,12 +2228,6 @@ SLJIT_API_FUNC_ATTRIBUTE sljit_s32 sljit_emit_mem(struct sljit_compiler *compile
 	sljit_s32 reg,
 	sljit_s32 mem, sljit_sw memw)
 {
-	SLJIT_UNUSED_ARG(compiler);
-	SLJIT_UNUSED_ARG(type);
-	SLJIT_UNUSED_ARG(reg);
-	SLJIT_UNUSED_ARG(mem);
-	SLJIT_UNUSED_ARG(memw);
-
```

Comment:
Please don't remove these. They are needed depending on macro configuration.

Reply:
got it

do you have a list of flags that you test with?; they could be make for a good thing to automate together with what I had been doing in carenas/sljit#6